src/domUpdates.js

```diff
@@ -66,26 +71,30 @@ let domUpdates = {
   },
 
   displayAvailableRooms(currentCustomer, roomsData, bookingsData) {
-    hide([greeting])
+    greeting.innerText = "Available Rooms"
     show([toBookDisplay])
     availableRoomsSection.innerHTML = '';
-    const date = selectedDate.value.split('-').join('/');
+    date = selectedDate.value.split('-').join('/');
     currentCustomer.getAvailableRooms(date, roomsData, bookingsData);
-    console.log('customer', currentCustomer)
 
-    console.log('customer', currentCustomer.availableRooms)
-    currentCustomer.availableRooms.forEach((room) => {
-      availableRoomsSection.innerHTML += `
-      <section class='individual-room-cards'>
+    if (currentCustomer.availableRooms.length > 0) {
```

Comment:
The apology is long and repeated. Granted, if it were just 'sorry' it would be a single line of code and not so obnoxious. But that's me- obnoxious.